2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.py

```diff
@@ -0,0 +1,22 @@
+# made by vanguard7
+def main():
+    t = int(input())
+    for i in range(t):
+        n = int(input())
+        low = 10000000000000000000000000000000000000000000
+        index = -1
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpdjftdys7/2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.py
+++ b/tmp/tmpdjftdys7/2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.py
@@ -19,4 +19,3 @@
 
 if __name__ == '__main__':
     main()
-    
```